Wikipedia/Code/WMFProxyServer.m

```diff
@@ -84,7 +91,10 @@ - (void)start {
 
     while (!didStartServer && attempts < attemptLimit) {
         didStartServer = [self.webServer startWithOptions:options error:&serverStartError];
```

Comment:
@joewalsh Looks like there's a [GCDWebServerOption_Port](https://github.com/swisspol/GCDWebServer/blob/master/GCDWebServer/Core/GCDWebServer.m#L53) option we could pass to `startWithOptions`.